File: output/ChatGPT/codenet/Java/C++/s461257670.cpp
#include <iostream>
using namespace std;

int Answer(int y, int m, int d) {
    int result = 1;

    if (y % 3 == 0) {
        result += 20 - d;
    } else if (m % 2 == 0) {
        result += 19 - d;
    } else {
        result += 20 - d;
    }

    for (m+=1; m <= 10; m++) {
        if (y % 3 == 0) {
            result += 20;
        } else {
            if (m % 2 == 0) {
                result += 19;
            } else {
                result += 20;
            }
        }
    }

    for (y += 1; y < 1000; y++) {
        if (y % 3 == 0) {
            result += 200;
        } else {
            result += 195;
        }
    }
    return result;
}

int main() {
    int count;
    cin >> count;
    for (int i = 0; count > i; i++) {
        int y, m, d;
        cin >> y >> m >> d;
        cout << Answer(y, m, d) << endl;
    }
}

File: output/CodeLlama/codenet/Java/Python/s461257670.py
import sys

def main():
    count = int(sys.stdin.readline())
    for i in range(count):
        y, m, d = map(int, sys.stdin.readline().split())
        print(answer(y, m, d))

def answer(y, m, d):
    result = 1
    if y % 3 == 0:
        result += 20 - d
    elif m % 2 == 0:
        result += 19 - d
    else:
        result += 20 - d
    for m in range(m+1, 11):
        if y % 3 == 0:
            result += 20
        elif m % 2 == 0:
            result += 19
        else:
            result += 20
    for y in range(y+1, 1000):
        if y % 3 == 0:
            result += 200
        else:
            result += 195
    return result

if __name__ == '__main__':
    main()